requirements.txt

```diff
@@ -29,3 +29,6 @@ black==21.5b2
 pre-commit==2.13.0
 flake8==3.9.2
 isort==5.9.1
+
+# for toxicity detection
```

Comment:
Please make a new `requirements.txt` file in this filter directory and move detoxify to that file. To make it not too heavy, the main requirements should only include dependencies that are used for most transformation/filters.

[Example](https://github.com/GEM-benchmark/NL-Augmenter/tree/main/transformations/close_homophones_swap)

Reply:
Sure thing